verify/terraform/skus/current/windows-11-23H2/Makefile

```diff
@@ -9,8 +9,8 @@ SHELL=/bin/bash
 all: apply
 
 terraform.tfvars:
-	cat ../../../modules/terraform.tfvars.template | sed -e "s/%password%/$$(pwgen -s --remove-chars=\'\"$$%{} -y 20 1)/gi" > terraform.tfvars
```

Comment:
I wrote the command line to remove some characters that cause problems when I run the generated `add_password.bat`. Is the changed version safe?

Reply:
Thank you for pointing out.
I suppose it will be the case and I have to revise that.

Reply:
Pushed a commit to fix